src/peft/utils/other.py

```diff
@@ -178,6 +180,36 @@ def _set_adapter(model, adapter_name):
             module.active_adapter = adapter_name
 
 
+def _is_matching_module_name(
```

Comment:
as mentioned in the above comment, regex matches for target module in case it is a str aren't being handled by this function